### ISSUE TITLE
Better user experience [Personal Experience backlog]

### ISSUE THREAD

**Issue by nicolay-r** [nicolay-r/AREnets#32]
- [x] `TermsPerContext` provide as a custom config callback modifier.
- [x] `Batch size` -- required for not to exceed the memory limit;
- [x] `Learning Rate` -- to modify the speed of training;
- [x] `model_input_type` -- (single, ctx)
- [x] Disable logging hidden states logging on training by default
- [x] Train `acc` limit move outside
- [x] Support input as the list of terms (#23)
- [x] `text_a` -> `text`
- [x] dropout 
- [x] remove `-0` (index) suffix for input filenames\
- [x] switch to English examples instead of Russian (due to internalization)
- [x] rename `rnn` to `lstm`
- [x] rename `att-bilstm` to `att-bilstm-pzhou`
- [x] **data description tutorial**!
- [x] `avg_acc` and `avg_loss`
